datatypes/src/jvmTest/kotlin/Pkcs10CertificationRequestJvmTest.kt

```diff
@@ -37,7 +37,7 @@ class Pkcs10CertificationRequestJvmTest : FreeSpec({
 
     "CSR match" {
         val ecPublicKey = keyPair.public as ECPublicKey
-        val cryptoPublicKey = CryptoPublicKey.Ec.fromJcaKey(ecPublicKey)
+        val cryptoPublicKey = CryptoPublicKey.Ec.fromJcaKey(ecPublicKey).getOrThrow()
         cryptoPublicKey.shouldNotBeNull()
```

Comment:
requires a `getOrNull()` in line 40 for `shouldNotBeNull()` to make sense

Reply:
True, will remove the shouldNotBeNull()